<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>815</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BNA Launcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="24" column="6">
      <widget class="QLineEdit" name="lineEditConvertNII_input_file"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelCUCorMat_Dir_for_BOLD">
       <property name="text">
        <string>Dir_for_BOLD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkCUCorMat">
       <property name="text">
        <string>CUCorMat</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLineEdit" name="lineEditCUCorMat_to_save_cormatrix"/>
     </item>
     <item row="0" column="1" colspan="6">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Construct correlation matrices from BOLD signals on GPU.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QToolButton" name="toolButtonCUCorMat_Dir_for_BOLD">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="lineEditCUCorMat_Dir_for_BOLD">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="labelCUCorMat_to_average">
       <property name="text">
        <string>to_average(y/n)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="7">
      <widget class="QToolButton" name="toolButtonCUBFW_Lp_input_dir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLabel" name="labelCUCorMat_threshold_for_mask">
       <property name="text">
        <string>threshold_for_mask</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLineEdit" name="lineEditCUCorMat_threshold_for_mask"/>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLabel" name="labelCUCorMat_to_save_cormatrix">
       <property name="text">
        <string>to_save_cormatrix(y/n)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="checkCUBFW_Lp">
       <property name="text">
        <string>CUBFW_Lp</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QLineEdit" name="lineEditCUBFW_Lp_input_dir"/>
     </item>
     <item row="3" column="6">
      <widget class="QLineEdit" name="lineEditCUCorMat_to_average"/>
     </item>
     <item row="7" column="1" colspan="7">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Calculate the characteristic path length of the network and compare with K (user specified) random networks. Based on blocked Floyd-Warshall algorithm (BFW).</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="5">
      <widget class="QLabel" name="labelCUBFW_Lp_num_of_random_networks">
       <property name="text">
        <string>num_of_random_networks</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="7">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Calculate the characteristic path length of the network and compare with K (user specified) random networks. Based on Breadth  First  Search (BFS).</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="labelCUBFW_Lp_input_dir">
       <property name="text">
        <string>input_dir</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="labelCUBFS_Lp_input_dir">
       <property name="text">
        <string>input_dir</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="5">
      <widget class="QLabel" name="labelCUBFS_Lp_num_of_random_networks">
       <property name="text">
        <string>num_of_random_networks</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="checkCUBFS_Lp">
       <property name="text">
        <string>CUBFS_Lp</string>
       </property>
      </widget>
     </item>
     <item row="9" column="6">
      <widget class="QLineEdit" name="lineEditCUBFW_Lp_num_of_random_networks"/>
     </item>
     <item row="18" column="1" colspan="5">
      <widget class="QLabel" name="labelCp_num_of_random_networks">
       <property name="text">
        <string>num_of_random_networks</string>
       </property>
      </widget>
     </item>
     <item row="17" column="7">
      <widget class="QToolButton" name="toolButtonCp_input_dir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="labelCp_input_dir">
       <property name="text">
        <string>input_dir</string>
       </property>
      </widget>
     </item>
     <item row="20" column="7">
      <widget class="QToolButton" name="toolButtonDegree_input_dir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="17" column="6">
      <widget class="QLineEdit" name="lineEditCp_input_dir"/>
     </item>
     <item row="20" column="1">
      <widget class="QLabel" name="labelDegree_input_dir">
       <property name="text">
        <string>input_dir</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QCheckBox" name="checkDegree">
       <property name="text">
        <string>Degree</string>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="QLabel" name="labelCUBC_input_dir">
       <property name="text">
        <string>input_dir</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1" colspan="6">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Calculate the degree centrality of the network.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="20" column="6">
      <widget class="QLineEdit" name="lineEditDegree_input_dir"/>
     </item>
     <item row="12" column="6">
      <widget class="QLineEdit" name="lineEditCUBFS_Lp_num_of_random_networks"/>
     </item>
     <item row="11" column="6">
      <widget class="QLineEdit" name="lineEditCUBFS_Lp_input_dir"/>
     </item>
     <item row="16" column="0">
      <widget class="QCheckBox" name="checkCP">
       <property name="text">
        <string>Cp</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1" colspan="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Calculate clustering coefficient (Cp) of the network and compare the results with  K  (user specified) random networks.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="7">
      <widget class="QToolButton" name="toolButtonCUBFS_Lp_input_dir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="18" column="6">
      <widget class="QLineEdit" name="lineEditCp_num_of_random_networks"/>
     </item>
     <item row="22" column="7">
      <widget class="QToolButton" name="toolButtonCUBC_input_dir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1" colspan="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Calculate the betweenness centrality of the network.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="22" column="6">
      <widget class="QLineEdit" name="lineEditCUBC_input_dir"/>
     </item>
     <item row="26" column="1" colspan="3">
      <widget class="QLabel" name="labelConvertNII_mask_threshold">
       <property name="text">
        <string>mask_threshold</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QCheckBox" name="checkCUBC">
       <property name="text">
        <string>CUBC</string>
       </property>
      </widget>
     </item>
     <item row="25" column="6">
      <widget class="QLineEdit" name="lineEditConvertNII_mask_file"/>
     </item>
     <item row="25" column="1">
      <widget class="QLabel" name="labelConvertNII_mask_file">
       <property name="text">
        <string>mask_file</string>
       </property>
      </widget>
     </item>
     <item row="24" column="1">
      <widget class="QLabel" name="labelConvertNII_input_file">
       <property name="text">
        <string>input_file</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QLineEdit" name="lineEditCUCorMat_threshold_for_correlation_coefficient"/>
     </item>
     <item row="30" column="0">
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save script</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="labelCUCorMat_threshold_type">
       <property name="text">
        <string>threshold_type(r/s)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLineEdit" name="lineEditCUCorMat_threshold_type"/>
     </item>
     <item row="26" column="6">
      <widget class="QLineEdit" name="lineEditConvertNII_mask_threshold"/>
     </item>
     <item row="30" column="1">
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="text">
        <string>Load script</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QCheckBox" name="checkBFS_MulCPU">
       <property name="text">
        <string>BFS_MulCPU</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="5">
      <widget class="QLabel" name="labelCUCorMat_threshold_for_correlation_coefficient">
       <property name="text">
        <string>threshold_for_correlation_coefficient</string>
       </property>
      </widget>
     </item>
     <item row="30" column="7">
      <widget class="QToolButton" name="toolButtonSaveDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QCheckBox" name="checkConvertNII">
       <property name="text">
        <string>ConvertNII</string>
       </property>
      </widget>
     </item>
     <item row="23" column="1" colspan="6">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Put the .cp .eloc .deg .bc results back to the 3-D matrix and converts these files to the standard NII format.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="27" column="0">
      <widget class="QCheckBox" name="checkL_Modularity">
       <property name="text">
        <string>L_Modularity</string>
       </property>
      </widget>
     </item>
     <item row="27" column="1" colspan="6">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Calculate Louvain Modularity for each of the .csr files in the given directory and compare with random networks.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="28" column="6">
      <widget class="QLineEdit" name="lineEditL_Modularity_dir_for_csr"/>
     </item>
     <item row="29" column="6">
      <widget class="QLineEdit" name="lineEditL_Modularity_num_of_random_networks"/>
     </item>
     <item row="29" column="1">
      <widget class="QLabel" name="labelL_Modularity_num_of_random_networks">
       <property name="text">
        <string>num_of_random_networks</string>
       </property>
      </widget>
     </item>
     <item row="28" column="1">
      <widget class="QLabel" name="labelL_Modularity_dir_for_csr">
       <property name="text">
        <string>dir_for_csr</string>
       </property>
      </widget>
     </item>
     <item row="15" column="6">
      <widget class="QLineEdit" name="lineEditBFS_MulCPU_num_of_random_networks"/>
     </item>
     <item row="14" column="6">
      <widget class="QLineEdit" name="lineEditBFS_MulCPU_input_dir"/>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="labelBFS_MulCPU_num_of_random_networks">
       <property name="text">
        <string>num_of_random_networks</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="labelBFS_MulCPU_input_dir">
       <property name="text">
        <string>input_dir</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="6">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Calculate the characteristic path length of the network and compare with K (user specified) random networks. Based on Breadth  First  Search (BFS) on multi-core CPU.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="28" column="7">
      <widget class="QToolButton" name="toolButtonL_Modularity_dir_for_csr">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="14" column="7">
      <widget class="QToolButton" name="toolButtonBFS_MulCPU_input_dir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="25" column="7">
      <widget class="QToolButton" name="toolButtonConvertNII_mask_file">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="24" column="7">
      <widget class="QToolButton" name="toolButtonConvertNII_input_file">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="30" column="6">
      <widget class="QLineEdit" name="lineEditSaveDir"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
